bot/cogs/tags.py

```diff
@@ -205,12 +225,14 @@ def _command_on_cooldown(tag_name: str) -> bool:
                         "time": time.time(),
                         "channel": ctx.channel.id
                     }
-                await ctx.send(embed=Embed.from_dict(tag['embed']))
+                msg = await ctx.send(embed=Embed.from_dict(tag['embed']))
+                await self.handle_trashcan_react(ctx, msg)
```

Comment:
Since object `msg` is not used anywhere else, you can combine the `send` and the `handle_trashcan_react()` together, and rename it to something like `send_with_thrashcan()` or something, that way we do not have to create a variable outside. Think of a wrapper on top of `ctx.send()` and we simply pass what we want to send to it.